scripts/checkpatch.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+SCRIPT=$(readlink -f $0)
+SCRIPT_PATH=$(dirname "$SCRIPT")
+CHECKPATCH=$SCRIPT_PATH/../../linux/scripts/checkpatch.pl
```

Comment:
I can change to something simple like that:
`hash checkpatch.pl 2>/dev/null || { echo >&2 "Could not find checkpatch.pl, aborting"; exit 1; }`

What do you think?